README.md

```diff
@@ -1,2 +1,61 @@
 # primo_ve
-UCLA Library Primo VE
+Primo VE Production
+
+This repo may be deployed to Primo VE at https://ucla.alma.exlibrisgroup.com/SAML.
+A *premium sandbox* (PSB), copied from production, is at https://sandbox02-na.alma.exlibrisgroup.com/institution/01UCS_LAL).
+
+Note: Deployment to either target is a manual process - this repo is used for source control.
+        This repo is defaulted for deployment to Production. Make the changes noted below for deployment to Test.
+
+**Workflow**
+
+Develop your code
+- clone primo_ve (this repo)
+- work, add, commit, push, merge
+- repeat until finished
+- ask for the secrets file ```secrets,json``` and place it in the ```primo_ve``` directory
```

Comment:
@darrowcoucla Minor typo in `secrets,json`

Reply:
Fixed typo.